.github/workflows/release-drafter.yml

```
@@ -0,0 +1,13 @@
+name: Release Drafter
+on:
+  push:
+  workflow_dispatch:
+  release:
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-20.04
```

```suggestion
    runs-on: ubuntu-22.04
```

(nit - self merging)